_posts/2018-02-02-rebble-web-services.md

```diff
@@ -0,0 +1,131 @@
+---
+layout: post
+title: Pebbling after Fitbit: introducing the Rebble Web Services
+date:   2018-02-02 11:42:42
+# categories: community
+---
+
+Hey there! [Katharine Berry](https://twitter.com/KatharineBerry) here. As you may well have heard
+by now, Fitbit will be shutting down all Pebble services at the end of June of this year. At that
+point, no services that require a Pebble server will work any more — which includes the appstore,
+weather, dictation, some parts of timeline, iOS SMS/email actions, the CloudPebble IDE, and more.
+
+The observant among you might also notice that none of these things have been mentioned in a
+community update in over a year. Fortunately, we have a plan: the Rebble Web Services.
+
+_(Disclaimer: absolutely everything in this post is subject to change)_
+
+------
+
+# Funding
```

Comment:
I don't really love putting this up top, but it makes the parts about charging people flow better. Ideas?

Reply:
I like the order but I would be more explicit or at least more clear (in that section) that if you donate you will get a reward (services). You put it more as a requirement ` and so for those we will be requiring that a small sum of money be pledged each month.` I think it's better to tell people it's a reward because those services are more expensive.

Reply:
Good point